servers/quarkus-server/build.gradle.kts

```diff
@@ -157,6 +157,12 @@ tasks.withType<Test>().configureEach {
   systemProperty("quarkus.smallrye.jwt.enabled", "true")
   // TODO requires adjusting the tests - systemProperty("quarkus.http.test-port", "0") -  set this
   //  property in application.properties
+  if (systemProperties["it.nessie.container.postgres.tag"] == null) {
```

Comment:
I doubt this does what it's meant to do.
Have you tried to run it locally and verify that it's really using a different version?

Reply:
original intention was to avoid overriding to "14", if user has set some value from command line using `-Dit.nessie.container.postgres.tag`

But not verified. @sagar-agarwal2404 : please test it and handle.